franka_joint_position_pd_gravity_controller:
    type: franka_controllers/JointPositionPDGravityController
    state_publish_rate: 100
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7

    q_d: [0, 0, 0, -1.57, 0, 1.57, 0.79]
    kd: [100, 100, 100, 100, 100, 100, 100]
    kp: [200, 200, 200, 200, 200, 200, 200]
    dtau_max: 1000 # set to -1 to disable rotatum saturation


